resolvers.js

```diff
@@ -526,6 +535,37 @@ const Resolvers = {
         logger.error(e.message);
         return e.message;
       }
+    },
+
+    async deleteOpticalImage(_, args) {
+      let {datasetId} = args;
+      const payload = jwt.decode(args.jwt, config.jwt.secret);
+      const basePath = `http://localhost:${config.img_storage_port}`;
+      try {
+        await checkPermissions(datasetId, payload);
+        let opticalImageIds = await pg.select('id').from('optical_image').where('ds_id', '=', datasetId);
+        let rawOpticalImage = await pg.select('optical_image').from('dataset').where('id','=',datasetId);
+        Array.prototype.push.apply(opticalImageIds, rawOpticalImage);
```

Comment:
I'd go for `concat` here to make it more readable

Reply:
Changed it, thanks @iprotsyuk 
Also seems that concat is a bit faster than Array.prototype.push.apply for Safari 
just fyi https://jsperf.com/array-prototype-push-apply-vs-concat/17